cmd/minishift/cmd/update.go

```diff
@@ -0,0 +1,45 @@
+/*
+Copyright (C) 2017 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cmd
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/minishift/minishift/pkg/minikube/update"
+	"github.com/minishift/minishift/pkg/util/os/atexit"
+
+	"github.com/spf13/cobra"
+)
+
+var updateCmd = &cobra.Command{
+	Use:   "update",
+	Short: "Checks for updates in Minishift",
+	Long:  `Checks for updates in Minishift and prompts the user to download newer version`,
+	Run:   runUpdate,
+}
+
+func runUpdate(cmd *cobra.Command, args []string) {
+	err := update.MaybeUpdateFromGithub(os.Stdout)
+	if err != nil {
+		atexit.ExitWithMessage(1, fmt.Sprintf("Update failed: %s", err))
+	}
+}
+
+func init() {
+	RootCmd.AddCommand(updateCmd)
```

Comment:
Sounds great. How about taking it as a separate issue/PR outside this one? I'll send in a new one once this gets merged.

Or do you think that as an absolute must for this PR to move further?